packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -211,20 +223,31 @@
           <button class="control-button diagram-reload" @click="resetDiagram" title="Clear">
             <ReloadIcon class="control-button__icon" />
           </button>
-          <v-popper
-            v-if="appMapUploadable"
-            class="hover-text-popper"
-            text="Create a link to this AppMap."
-            placement="left"
-            text-align="right"
-          >
-            <button class="control-button appmap-upload" @click="uploadAppmap" title="">
-              <UploadIcon class="control-button__icon" />
-              <span>Share</span>
-            </button>
-          </v-popper>
         </template>
       </v-tabs>
+
+      <div v-if="showShareModal" class="share-appmap">
+        <div class="heading">
+          <h1>Share this AppMap</h1>
+          <span class="close-me" @click="closeShareModal">X</span>
```

Comment:
These also exist:
- https://github.com/getappmap/appmap-js/blob/main/packages/components/src/assets/close.svg
- https://github.com/getappmap/appmap-js/blob/main/packages/components/src/assets/close-thin.svg

I'm not sure which is most appropriate, but we're definitely using these elsewhere.

Reply:
Thanks! I switched to use the `close.svg`.